.github/workflows/nox.yml

```diff
@@ -29,27 +29,27 @@ jobs:
       matrix:
         include:
           - session: test
-            python-versions: "3.9, 3.10, 3.11"
+            python-versions: "3.9, 3.10, 3.11, 3.12"
             codecov: true
             packages: ""
 
           - session: "check_package_files(8.1.0)"
-            python-versions: "3.11"
+            python-versions: "3.12"
```

Comment:
This doesn't work as you expect. The current version of the nox action always uses `python3.11` as the base python and doesn't let you change. I can change this to roll our own thing without that limitation if you'd like.

Reply:
That's kind of annoying. I've reverted this in ff989abb888dcfe295c5993aade32e2df84815a3.